src/components/GlobalPreferences/GlobalPreferences.js

```diff
@@ -0,0 +1,208 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import {
+  Bar,
+  ButtonIcon,
+  GU,
+  Header,
+  IconClose,
+  Layout,
+  Tabs,
+  breakpoint,
+  springs,
+  textStyle,
+  useTheme,
+  useViewport,
+} from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { useEsc } from '../../hooks/useKeyboardArrows'
+import Network from './Network/Network'
+
+const SECTIONS = new Map([['network', 'Network']])
+const PATHS = Array.from(SECTIONS.keys())
+const VALUES = Array.from(SECTIONS.values())
+
+const NETWORK_INDEX = 0
+
+const AnimatedDiv = animated.div
+
+function GlobalPreferences({ compact, onClose, onNavigation, sectionIndex }) {
+  const theme = useTheme()
+
+  useEsc(onClose)
+
+  const tabItems = VALUES.filter((_, index) => index === NETWORK_INDEX)
+
+  const container = useRef()
+  useEffect(() => {
+    if (container.current) {
+      container.current.focus()
+    }
+  }, [])
+
+  return (
+    <div ref={container} tabIndex="0" css="outline: 0">
+      <Layout css="z-index: 2">
+        <Close compact={compact} onClick={onClose} />
+        <Header
+          primary="Global preferences"
+          css={`
+            padding-top: ${!compact ? 10 * GU : 0}px;
+          `}
+        />
+        <React.Fragment>
+          {tabItems.length > 1 ? (
+            <Tabs
+              items={tabItems}
+              onChange={onNavigation}
+              selected={sectionIndex}
+            />
+          ) : (
+            <Bar>
+              <div
+                css={`
+                  display: flex;
+                  height: 100%;
+                  align-items: center;
+                  padding-left: ${compact ? 2 * GU : 3 * GU}px;
+                  color: ${compact
+                    ? theme.surfaceContent
+                    : theme.surfaceContentSecondary};
+                  ${textStyle('body2')}
+                `}
+              >
+                {tabItems[0]}
+              </div>
+            </Bar>
+          )}
+          <main>{sectionIndex === NETWORK_INDEX && <Network />}</main>
+        </React.Fragment>
+      </Layout>
+    </div>
+  )
+}
+
+function useGlobalPreferences({ path = {}, onScreenChange }) {
```

Comment:
```suggestion
function useGlobalPreferences({ path = '', onScreenChange }) {
```

It looks like `path` is used directly like a string here.